app/controllers/coronavirus_controller.rb

```diff
@@ -37,6 +37,14 @@ def publish
     redirect_to coronavirus_path(slug)
   end
 
+  def manage_page
```

Comment:
Would you mind just deleting these two actions then feel free to merge!

Reply:
thank you @hannako, will be merging as soon as the checks are finished  🚀